HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -1305,7 +1305,7 @@ def self.set_cool_curves_room_ac(cooling_system)
     clg_ap.cool_cap_ft_spec = [[0.43945980246913574, -0.0008922469135802481, 0.00013984567901234569, 0.0038489259259259253, -5.6327160493827156e-05, 2.041358024691358e-05]]
     clg_ap.cool_eir_ft_spec = [[6.310506172839506, -0.17705185185185185, 0.0014645061728395061, 0.012571604938271608, 0.0001493827160493827, -0.00040308641975308644]]
     clg_ap.cool_cap_fflow_spec = [[0.887, 0.1128, 0]]
```

Comment:
I assume we can also replace the capacity curves by 1.0? Even though it is just slightly off 1 in the simulation.

Reply:
Good idea, I'll do that. 

I also noticed we're using [1, 0, 0] for mini-splits (for both eri_fflow and cap_fflow), so room ACs are now consistent.